src/In.ProjectEKA.HipService/Discovery/PatientDiscovery.cs

```diff
@@ -74,9 +81,12 @@ public class PatientDiscovery
                 return (null, error);
             }
 
+            var patient = patients.First(info => info.Identifier == patientEnquiryRepresentation.ReferenceNumber);

```

Comment:
Codacy found an issue: [Remove the unused local variable 'patient'.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5811615)